test/docs/transactions.test.js

```diff
@@ -338,6 +338,26 @@ describe('transactions', function() {
     assert.deepEqual(fromDb, { name: 'Tyrion Lannister' });
   });
 
+  it('distinct (gh-8006)', async function() {
+    const Character = db.model('gh8006_Character', new Schema({ name: String, rank: String }, { versionKey: false }));
+
+
+    const session = await db.startSession();
```

Comment:
```suggestion
    const Character = db.model('gh8006_Character', new Schema({ name: String, rank: String }, { versionKey: false }));

    const session = await db.startSession();
```

i dont know if this intended, but i think a single new line would be enough